README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
```

Comment:
@Logan1x In the screenshot form:

![screenshot from 2017-07-23 19 04 53](https://user-images.githubusercontent.com/16105680/28499718-e85ea702-6fd9-11e7-97cf-b73079704d3a.png)

![screenshot from 2017-07-23 19 06 00](https://user-images.githubusercontent.com/16105680/28499720-002fd432-6fda-11e7-8daf-2994312ce2fa.png)